src/sanitizer.js

```diff
@@ -320,6 +339,12 @@ export function isValidAttr(tagName, attrName, attrValue) {
     return false;
   }
 
+  if (attrName == 'class' &&
+      attrValue &&
+      /(^|\W)i-amphtml-/i.test(attrValue)) {
+    return false;
+  }
```

Comment:
Not ideal since we're duping that regex again. Please add a link to the validator protoascii in a comment.